crescent/ext/tasks/__init__.py

```diff
@@ -0,0 +1,7 @@
+from typing import Sequence
+
+from crescent.ext.tasks.cron import *
+from crescent.ext.tasks.loop import *
+from crescent.ext.tasks.task import *
+
+__all__: Sequence[str] = ("cronjob", "loop", "TaskCallbackT")
```

Comment:
`loop` feels too generic.

Reply:
I can see that the file can be confused with Task. I dont want to change the function `loop`'s name because `task.loop` is very clear imo.